components/Dialogs/src/private/MDCAlertControllerView+Private.m

```diff
@@ -255,6 +255,15 @@ - (void)setButtonInkColor:(UIColor *)color {
   }
 }
 
+- (CGFloat)cornerRadius {
+  return self.layer.cornerRadius;
+}
+
+- (void)setCornerRadius:(CGFloat)cornerRadius {
+  self.layer.cornerRadius = cornerRadius;
```

Comment:
Suggestion/optimization: Please use `MDCCGFloatEqual(CGFloat, CGFloat)` to check if the new and old values are different. If they are the same, let's return early and avoid calling `setNeedsLayout`.